mesh/mesh_readers.cpp

```diff
@@ -685,10 +1050,184 @@ void Mesh::ReadVTKMesh(std::istream &input, int &curved, int &read_gf,
             }
          }
       }
+         points.Destroy();
```

Comment:
Maybe some more whitespaec weirdness.  You should be able to run make style on the branch to fix stuff like this up.